ganga/GangaCore/Lib/Executable/Executable.py

```diff
@@ -288,3 +288,28 @@ def prepare(self, app, appconfig, appmasterconfig, jobmasterconfig):
 allHandlers.add('Executable', 'CREAM', LCGRTHandler)
 allHandlers.add('Executable', 'ARC', LCGRTHandler)
 allHandlers.add('Executable', 'Slurm', RTHandler)
+
+def randomString():
```

Comment:
*I realised the previous comment didn't really show the whole code so more useful to see the function prototype*